I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 150, in script
    show grimm at right with act
  File "game/script.rpy", line 150, in <module>
    show grimm at right with act
NameError: name 'act' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 150, in script
    show grimm at right with act
  File "C:\Program Files (x86)\RenPy\renpy\ast.py", line 1674, in execute
    paired = renpy.python.py_eval(self.paired)
  File "C:\Program Files (x86)\RenPy\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Program Files (x86)\RenPy\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 150, in <module>
    show grimm at right with act
NameError: name 'act' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.1.23060707
Blood Knight 0.29
Tue Aug 15 15:44:59 2023
